motr/m0crate/parser.c

```diff
@@ -535,6 +537,8 @@ int copy_value(struct workload *load, int max_workload, int *index,
 		case ADDB_INIT:
 			conf->is_addb_init = atoi(value);
 			break;
+		case ADDB_SIZE:
+			conf->addb_size = atoi(value);
```

Comment:
This might be:
```
conf->addb_size = atol(value);
```

Please test a large number: e.g. 10G.
I think 10G is overflow an "int".

Reply:
Done...1bd582e
Removed atoi(). With new chnage user can supply both human readable(g,G,m,M etc) and normal input sizes in bytes.